lib/sass_spec/engine_adapter.rb

```diff
@@ -111,7 +119,9 @@ def version
   end
 
   def compile(sass_filename, precision)
-    @stdin.puts "--no-color --no-unicode #{@args} #{sass_filename}"
+    dirname, basename = File.split(sass_filename)
+    @stdin.puts "!cd #{File.absolute_path(dirname)}"
```

Comment:
I'm not sure what `!cd` is but it may be necessary to shell-escape the path (`Shellwords.escape`)

Reply:
It isn't; it's just used as a signal to the inner process that this is the directory to move to. See [these lines](https://github.com/sass/sass-spec/blob/453385d2a400fdaeb3b44b3821f67d20405c521d/lib/sass_spec/engine_adapter.rb#L85-L88).